src/peg/mod.rs

```diff
@@ -23,7 +20,7 @@ mod tests;
 #[derive(Debug, PartialEq)]
 pub(crate) enum PegBody {
     /// A basic expression.
-    BasicOp(Expr<Id>),
+    BasicOp(RvsdgExpr<Id>),
```

Comment:
We're still using this for PEGs, right? should we name this something else?

Also, to Anjali's point earlier, this sort of rename is really easy to review if it's the entirety of the PR (w/the remaining changes stacked on top).

Reply:
Oh I didn't realize it was used in PEGs.
Yeah, renaming should be separate I agree.
I'll do another PR for that!